src/main/resources/messages/messages.properties

```diff
@@ -305,6 +305,7 @@ error.crowdin_not_support_file_format='%s' file format is not supported for targ
 error.file_already_exists=Project already contains the file '%s'
 error.uploading_file=File '%s'
 error.duplicate_environment_variable=There might be a duplicate environment variable(possibly not related to Crowdin CLI) in your PATH or .env file
+error.files.is.empty=Files is empty, Please provide files.
```

Comment:
error.no_files_found_for_pre_translate=Couldn't find any files to Pre-Translate in the current project